src/pages/Create.js

```diff
@@ -0,0 +1,424 @@
+                           onChange={handleTitleChange}
+                           InputProps={{style: {fontSize: 40,marginBottom: 12}}}
+                />
+            </Grid>
+            <Grid container direction={'row'} alignItems={'center'}>
+                <TextField label={'Subtitle'} fullWidth={true} multiline={true} rowsMax={4}
+                           onChange={handleSubtitleChange}
+                           onKeyDown={(event) => {
+                               if (event.key==='Tab'){
+                                   setTimeout(() => {
+                                       ReactEditor.focus(editor);
+                                   }, 100);
+                               }
+                           }}
+                           InputProps={{style: {fontSize: 30,marginBottom: 12}}}
+                />
+            </Grid>
+            <Slate editor={editor}
+                   value={blogBody}
+                   onChange={newValue => setBlogBody(newValue)}
+                   >
+                <HoveringToolbar />
+                <Editable
+                    renderElement={renderElement}
+                    renderLeaf={renderLeaf}
+                    placeholder={'Tell your story...'}
+                    onKeyDown={event => {
+                        if (event.metaKey) {
+                            switch (event.key) {
+                                case '`': {
+                                    event.preventDefault();
+                                    formatBlock(editor, 'code');
+                                    break
+                                }
+                                case 'b': {
+                                    event.preventDefault();
+                                    toggleFormat(editor, 'bold');
+                                    break
+                                }
+                                case 'i': {
+                                    event.preventDefault();
+                                    toggleFormat(editor, 'italic')
+                                    break
+                                }
+                                case 'u': {
+                                    event.preventDefault();
+                                    toggleFormat(editor, 'underlined');
+                                    break
+                                }
+                            }
+                        } else if (event.key==='Tab') {
+                            event.preventDefault();
+                            Editor.insertText(editor, '    ');
+                        }
+                    }}
+                    onKeyUp={event => {
+                        if (event.key==='Enter') {
+                            formatBlock(editor,'paragraph');
+                        }
+                    }}
+                />
+            </Slate>
+
+
+
+        </Container>
+    );
+}
+
+const CodeElement = props => {
```

Comment:
Same thought on the future editing blog.

Will all these end up being common between create and edit? Should we refactor them now?